src/test/java/org/openmrs/performance/http/HttpService.java

```diff
@@ -115,4 +126,32 @@ public HttpRequestActionBuilder getActiveOrders(String patientUuid) {
 		return http("Get Active Orders")
 				.get("/openmrs/ws/rest/v1/order?patient="+patientUuid+"&careSetting=6f0c9a92-6f24-11e3-af88-005056821db0&status=ACTIVE&orderType=131168f4-15f5-102d-96e4-000c29c2a5d7&v=custom:(uuid,dosingType,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,encounter:ref,orderer:(uuid,display,person:(display)),orderReason,orderReasonNonCoded,orderType,urgency,instructions,commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)");
 	}
+
+	public HttpRequestActionBuilder saveVitalsData(String patientUuid, int value) {
+		ZonedDateTime now = ZonedDateTime.now();
+		String encounterDatetime = now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ"));
+
+		Map<String, Object> saveVitals = new HashMap<>();
+		saveVitals.put("form", VISIT_NOTE_FORM_UUID);
+		saveVitals.put("patient", patientUuid);
+		saveVitals.put("location", OUTPATIENT_CLINIC_LOCATION_UUID);
+		saveVitals.put("encounterType", VISIT_NOTE_ENCOUNTER_TYPE_UUID);
+		saveVitals.put("encounterDatetime", encounterDatetime);
+		
+		Map<String, Object> obs = new HashMap<>();
+		obs.put("concept", Map.of("uuid", VITALS_CONCEPT));
+		obs.put("value", value);
+		
+		saveVitals.put("obs", List.of(obs));
```

Comment:
A single submission contains all the vitals (and biometrics) in it as a list of obs. ex: temperature is one observation. 
ex: `5085AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA` (the value corresponding to the `VITALS_CONCEPT ` you defined) concept id is systolic Blood Pressure. 

Full list of concept ids can be found here:

https://github.com/openmrs/openmrs-esm-patient-chart/blob/1bb5ccbc827def608924545747d55d0fe2c67b66/packages/esm-patient-vitals-app/src/config-schema.ts#L3

Reply:
Thank @jayasanka-sack  for reviewing my PR. I will make the necessary changes